docs/install.rst

```diff
@@ -14,153 +14,148 @@ Once installation is complete, you can run :ref:`mitmproxy`, :ref:`mitmdump` or
 Installation on macOS
 ---------------------
 
-You can use Homebrew to install everything:
+The recommended way to install mitmproxy on macOS is to use `Homebrew`_:
 
 .. code:: bash
 
     brew install mitmproxy
 
-Or you can download the pre-built binary packages from our `releases`_.
+Alternatively you can download our self-contained pre-built binary packages from
+our `releases`_ page.
 
 
 .. _install-windows:
 
 Installation on Windows
 -----------------------
 
-The recommended way to install mitmproxy on Windows is to use the installer
-provided at `mitmproxy.org`_. After installation, you'll find shortcuts for
-:ref:`mitmweb` (the web-based interface) and :ref:`mitmdump` in the start menu.
-Both executables are  added to your PATH and can be invoked from the command
-line.
+The recommended way to install mitmproxy on Windows is to download our
+self-contained pre-built binary packages from our `releases`_ page.
+
+After installation, you'll find shortcuts for :ref:`mitmweb` (the web-based
+interface) and :ref:`mitmdump` in the start menu. Both executables are added to
+your PATH and can be invoked from the command line.
 
 .. note::
-    Mitmproxy's console interface is not supported on Windows, but you can use
-    mitmweb (the web-based interface) and mitmdump.
+    The console interface `mitmproxy` is not supported on Windows, but you can
+    use mitmweb (the web-based interface) and mitmdump.
+
 
 .. _install-linux:
 
 Installation on Linux
 ---------------------
 
-The recommended way to run mitmproxy on Linux is to use the pre-built binaries
-provided at `releases`_.
-
-Our pre-built binaries provide you with the latest version of mitmproxy, a
-self-contained Python 3.5 environment and a recent version of OpenSSL that
-supports HTTP/2. Of course, you can also install mitmproxy from source if you
-prefer that (see :ref:`install-advanced`).
-
-.. _install-advanced:
+The recommended way to install mitmproxy on a recent Linux distribution (with
+Python 3.5 or higher) is to use pip3:
```

Comment:
Why pip > prebuilt binaries? I think I'm -1 on this. The portable binaries avoid all the pip/Python version/Python package management issues listed here. For the presumably small set of mitmproxy users that require custom packages in their inline scripts, re-installing using pip seems ok to me.

(Besides, this would also make it much more difficult to upgrade to Python 3.6 for example)

Reply:
I personally am in favour of promoting pip as default installation method.

However, to ease future Python upgrades and reduce dependency / version conflicts, I will revert this change to again recommend our binary packages over pip.